src/main/java/streams/Main.java

```diff
@@ -17,36 +17,52 @@ public static void main(String[] args) {
         File inputFile = new File("src/main/java/streams/KeyContaining.java");
         File outputFile = new File("src/main/java/streams/key.txt");
 
-        try (FileInputStream inputStream = new FileInputStream(inputFile);
-             FileOutputStream outputStream = new FileOutputStream(outputFile);
-             BufferedInputStream bufferedInputStream = new BufferedInputStream(inputStream);
-             BufferedOutputStream bufferedOutputStream = new BufferedOutputStream(outputStream)) {
+        try (FileReader fileReader = new FileReader(inputFile);
+             FileWriter fileWriter = new FileWriter(outputFile);
+             BufferedReader bufferedReader = new BufferedReader(fileReader);
+             BufferedWriter bufferedWriter = new BufferedWriter(fileWriter)) {
 
-            Map<String, Integer> keywordCount = countKeywords(bufferedInputStream);
-            writeKeywordsCount(bufferedOutputStream, keywordCount);
+            Map<String, Integer> keywordCount = countKeywords(bufferedReader);
+            writeKeywordsCount(bufferedWriter, keywordCount);
 
         } catch (IOException e) {
             System.out.println("Error" + e.getMessage());
         }
     }
 
+    public static Map<String, Integer> countKeywords(BufferedReader bufferedReader) {
+        StringBuilder content = new StringBuilder();
+        try {
+            String line;
+            while ((line = bufferedReader.readLine()) != null) {
+                content.append(line).append("\n");
+            }
+
+        } catch (IOException e) {
+            System.out.println("Error" + e.getMessage());
```

Comment:
consider adding context, like "Error reading file: " or "Error writing file: " to improve debugging.